### PR TITLE
Vv 76 mssql integration

### PR DESCRIPTION
**Pull request by pl-madetech**
# What
Add MSSQL module. Fix scripts.

# Why
Replace postgres with mssql.

# Screenshots

# Notes

### CODE REVIEW

File: docs/development/README.md [hunk 1]
```diff
@@ -57,6 +72,27 @@ SENTRY_AUTH_TOKEN=
...
+Set in your `.env` file the following properties.
+
+MSQL_DB_DRIVER=mssql\
+MSQL_DB_PORT=1433\
+MSQL_DB_POOL_MAX=15\
+MSQL_DB_POOL_MIN=5\
+MSQL_DB_POOL_IDLE_TIMEOUT=30000\
+MSQL_DB_USER=sa\
+MSQL_DB_PASSWORD=P@55w0rd\
+MSQL_DB_SERVER=localhost\
```
Comment (miranda-hawkes) [inline docs/development/README.md:88]: I assume these need `\` removing, caused an error on my machine

**Review by miranda-hawkes — CHANGES_REQUESTED** [review]
Two tiny comments as well as merge conflicts

File: docs/development/README.md (hunk 1 above)
Comment (pl-madetech) [reply docs/development/README.md:88]: It's a line break on the .md will check how to do it without is or adding a space

File: docs/development/README.md [hunk 2]
```diff
@@ -57,6 +72,27 @@ SENTRY_AUTH_TOKEN=
...
+MSQL_DB_DATABASE=nhs_virtual_visit_dev
+
+Run `sudo ./bin/setup_local_mssql_docker.sh`
```
[the remark this answers is not in the document]
Comment (pl-madetech) [reply docs/development/README.md:91]: yeah miss that one. will remove

File: docs/development/README.md [hunk 3]
```diff
@@ -57,6 +72,32 @@ SENTRY_AUTH_TOKEN=
...
+
+<<<<<<< HEAD
```
Comment (miranda-hawkes) [inline docs/development/README.md:93]: Some rebasing lines in here sorry!

**Comment by pl-madetech** [comment]
Another branch with rebase was merged.

**Pull request closed without merging**